policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+# Project Director Election Process
+
+## Aims
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria, focusing on shared power and new leadership rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity, aiming for the overlap of tolerance rather than the overlap of preference.
+* Conduct the selection process in a time-efficient manner.
+* Incorporate input from a wide range of stakeholders across the entire Rust project.
+* Ensure a diverse pool of qualified candidates for selection.
+* Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than preference or criticism.
+* Ensure participants feel positive about their interactions and the execution of the process.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
+    * **Aims of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review or Define Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944))
+    * Aims
+    * Time Investment
+    * Responsibilities
+    * Qualifications
+    * Term
+
+### Pre-Meeting Coordination
+
+* Gather nominations
+    * Council representatives collaborate with their subteams to collect nominations on behalf of their branch of the organization tree.
+    * Subteams can utilize any process they prefer, but we recommend following a similar approach to the process outlined in the "Election Meeting" section below
+* Share nominations publicly
+    * Each team representative must publicly share their team's nominations at least ten days before the final election meeting. This allows other teams to share feedback or objections about candidates their team didn't initially consider with their representative.
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting
+
+### Election Meeting
```

Comment:
Is it actually mentioned anywhere who is participating in this meeting?

Reply:
I would put it directly below this heading:

> The election meeting is compromised of all members of the Leadership Council who will actively participate in the consensus process as well as the facilitator who will facilitate that process.

This also makes me think that we explicitly don't want the facilitator to be a Council Member to avoid someone having to split their mental capacity across two tasks.